modules/ROOT/pages/dw-core-functions-reduce.adoc

```diff
@@ -19,13 +19,13 @@ accumulator, a null value is returned.
 |===
 | Name   | Description
 | `element` | The current element from the array. Can also be referenced as `&#36;`. If this parameter is not named in the lambda expression.
-| `acc` | The accumulator. Can also be referenced as `&#36;&#36;`. Used to store the result of the lambda expression after each iteration of the reduce operation. 
+| `acc` a| The accumulator. Can also be referenced as `&#36;&#36;`. Used to store the result of the lambda expression after each iteration of the reduce operation.
 
-The accumulator parameter can be set to an intial value using the syntax `acc = initValue`. In this case the lambda expression is called with the first element of the input array, then the result is set as the new accumulator value. 
+The accumulator parameter can be set to an intial value using the syntax `acc = initValue`. In this case the lambda expression is called with the first element of the input array, then the result is set as the new accumulator value.
```

Comment:
In this case, ...
"and then..." instead of "then" (I know this is a style thing but it sounds better to me.)